### ISSUE TITLE
Images of current results are rotated

### ISSUE THREAD

**Issue by yeguixin** [junyanz/pytorch-CycleGAN-and-pix2pix#55]
When I view the current results, I find that the images are rotated. Is it okay for the software?

**Comment by junyanz** [comment]
The images should not be rotated. Did you try the code on our datasets or your data? Could you share a screenshot?

**Comment by yeguixin** [comment]
I run the code using my datasets. The current results are as  follows:
![current_result](https://user-images.githubusercontent.com/16981269/27891620-627a8836-622d-11e7-8802-8522acc5a453.png)

Normally, It should be as shown on epoch[17]. But on epoch[16], it seems to be mirror rotated. The dataset is okay. Thank you.

**Comment by taesung89** [comment]
The training images are horizontally flipped with 50% probability. If you want to prevent this, please specify --no_flip as one of the options. 

Did you observe flipping or rotation in any other direction?

**Comment by yeguixin** [comment]
Yes, they are horizontally flipped and no other directions. Thank you very much.
In addition, I want to use pix2pix to train a model which aims to transform the the image shown on real_A (epoch 17)to the image shown on real_B (epoch 17). Do you think pix2pix model can do it according to your rich experience? In recent days, I trained some models using different options such as different batchSize and learning rate. But the result is much worse than expected. It seems that the model does not converge because the value of G_GAN has been fluctuating. Thanks for your advise.

**Comment by junyanz** [comment]
I think it is possible as people have used pix2pix for generating Chinese Calligraphy. See [zi2zi](https://github.com/kaonashi-tyc/zi2zi) project for more details. I will recommend the default learning rate and batchSize as a starting point.

**Comment by yeguixin** [comment]
Got it. I increased the number of training sets to 50,000 and the result become better than previous. Thank you very much.
